ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -30,31 +30,34 @@ open class CommandBar: UIView {
     // MARK: - Public methods
 
     @available(*, renamed: "init(itemGroups:leadingItemGroups:trailingItemGroups:)")
-    @objc public convenience init(itemGroups: [CommandBarItemGroup], leadingItem: CommandBarItem? = nil, trailingItem: CommandBarItem? = nil) {
+    @objc public convenience init(itemGroups: [CommandBarItemGroup],
+                                  leadingItem: CommandBarItem? = nil,
+                                  trailingItem: CommandBarItem? = nil) {
         var leadingItems: [CommandBarItemGroup]?
         var trailingItems: [CommandBarItemGroup]?
 
         if let leadingItem = leadingItem {
             leadingItems = [[leadingItem]]
         }
-
         if let trailingItem = trailingItem {
             trailingItems = [[trailingItem]]
         }
 
-        self.init(itemGroups: itemGroups, leadingItemGroups: leadingItems, trailingItemGroups: trailingItems)
+        self.init(itemGroups: itemGroups,
```

Comment:
Small suggestion for making this code a bit more Swifty:

You can declare constants (let) and initialize them with closures. Something like below:

```
    @available(*, renamed: "init(itemGroups:leadingItemGroups:trailingItemGroups:)")
    @objc public convenience init(itemGroups: [CommandBarItemGroup],
                                  leadingItem: CommandBarItem? = nil,
                                  trailingItem: CommandBarItem? = nil) {
        let leadingItems: [CommandBarItemGroup]? = {
            guard let leadingItem = leadingItem else {
                return nil
            }

            return [[leadingItem]]
        }()

        let trailingItems: [CommandBarItemGroup]? = {
            guard let trailingItem = trailingItem else {
                return nil
            }

            return [[trailingItem]]
        }()

        self.init(itemGroups: itemGroups,
                  leadingItemGroups: leadingItems,
                  trailingItemGroups: trailingItems)
    }
```

Reply:
This is great!